static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -124,6 +133,28 @@ function DefaultGroupEventDetailsContent({
           />
         </EventDataSection>
       )}
+      {event.type === EventOrGroupType.ERROR &&
+      organization.features.includes('ai-analytics') &&
+      event?.entries
+        ?.filter((x): x is EntryException => x.type === EntryType.EXCEPTION)
+        .flatMap(x => x.data.values ?? [])
+        .some(({value}) => {
+          const lowerText = value.toLowerCase();
+          return (
+            (lowerText.includes('api key') || lowerText.includes('429')) &&
+            (lowerText.toLowerCase().includes('openai') ||
+              lowerText.toLowerCase().includes('anthropic') ||
+              lowerText.toLowerCase().includes('cohere') ||
+              lowerText.toLowerCase().includes('langchain'))
+          );
+        }) ? (
+        <LazyLoad
+          LazyComponent={AIMonitoringSection}
```

Comment:
you'll need to import this differently, https://github.com/getsentry/sentry/blob/1b9c454ce7987fd02a114da8e7c1a6d1fc856cdc/static/app/components/events/rrwebIntegration.tsx#L22 it will also need to be a default export instead of `AIMonitoringSection` to make this easier